website/source/docs/providers/azurerm/r/express_route_circuit.html.markdown

```diff
@@ -0,0 +1,80 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_express_route_circuit"
+sidebar_current: "docs-azurerm-resource-express-route-circuit"
+description: |-
+  Creates an ExpressRoute circuit.
+---
+
+# azurerm\_express\_route\_circuit
+
+Creates an ExpressRoute circuit.
+
+## Example Usage
+
+```hcl
+resource "azurerm_express_route_circuit" "test" {
+  name                     = "expressRoute1"
+  resource_group_name      = "${azurerm_resource_group.test.name}"
+  location                 = "West US"
+  service_provider_name    = "Equinix"
+  peering_location         = "Silicon Valley"
+  bandwidth_in_mbps        = 50
+  sku_tier                 = "Standard"
+  sku_family               = "MeteredData"
+  allow_classic_operations = false
+
+  tags {
+    environment = "Production"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the ExpressRoute circuit. Changing this forces a
+    new resource to be created.
+
+* `resource_group_name` - (Required) The name of the resource group in which to
+    create the namespace. Changing this forces a new resource to be created.
+
+* `location` - (Required) Specifies the supported Azure location where the resource exists.
+    Changing this forces a new resource to be created.
```

Comment:
I guess you need the silly "Changing this forces a new resource to be created." everywhere you use ForceNew?